legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/gitlab/auth/AbstractTestGitLabSession.java

```diff
@@ -42,41 +43,48 @@ public void testEncoding_Empty()
     public void testEncoding_OneAccessToken()
```

Comment:
Make sure we have some tests in this class that involve private access tokens. Ideally, we should have some that are just OAuth tokens, some that are just private access tokens, and some that are a mix.